client_test.go

```diff
@@ -0,0 +1,21 @@
+package helmclient
+
+import (
+	"testing"
+
+	"github.com/golang/mock/gomock"
+)
+
+func TestUpdateChartRepos(t *testing.T) {
+	var e error
+	ctrl := gomock.NewController(t)
+	defer ctrl.Finish()
+
+	mockClient := NewMockClient(ctrl)
+
+	mockClient.EXPECT().
+		UpdateChartRepos().
+		Return(e).
+		Times(1).
+		Do(mockClient.UpdateChartRepos())
```

Comment:
This unit tests effectively tests the `gomock` package. Maybe rewrite it to an example function that shows how to use the mock client?